javascript/ql/src/Security/CWE-798/HardcodedCredentials.ql

```diff
@@ -22,8 +22,14 @@ where
   // use source value in message if it's available
   if source.getNode().asExpr() instanceof ConstantString
   then
-    value = "The hard-coded value \"" + source.getNode().asExpr().(ConstantString).getStringValue() +
-        "\""
+    exists(string val | val = source.getNode().asExpr().(ConstantString).getStringValue() |
```

Comment:
I think this can be simplified to `source.getNode().getStringValue()`.

Reply:
Fixed